### ISSUE TITLE
Incomplete message threads

### ISSUE THREAD

**Issue by restrainedimages** [fetlife/fetlife-android#25]
If I open an existing conversation and scroll back then at some point it stops. I know that isn't the start of the conversation so it isn't pulling back the rest.

This could be a design decision. But if so I'd make it clear there is earlier text that isn't displayed.
I suspect it is only pulling so much and isn't detecting when you reach the top of that to pull the next block.

As a short term fix I just think it needs to be clear.
As a long term fix then it would be nice if you could scroll back through the whole conversation.

**Comment by kistitan** [comment]
Hi there,

Thanks very much for your feedback.
No, of course it is not a design decision :) Messages will be able to scrolled back in time just like at the homepage.
It is currently only a missing feature.

The app is still in a preliminary state as there are still some essential features missing. (For example paging for messaging and conversations and starting new conversation).

We decided to open the app before we reached the minimal viable product to be transparent for the community and to get feedbacks as soon as it is possible.
And we are working hard to add the most important features like the one you referring too :)

I will keep you updated via this thread about the upcoming releases and the state of this missing feature :)
